src/pe/section_table.rs

```diff
@@ -93,7 +92,25 @@ impl SectionTable {
         if idx <= 9_999_999 { // 10^7 - 1
             self.name = [0; 8];
             self.name[0] = b'/';
-            write!(&mut self.name[1..], "{}", idx).unwrap();
+            if idx == 0 {
```

Comment:
i literally have no idea what this is doing :laughing:

Reply:
It's doing `write!(&mut self.name[1..], "{}", idx)` without using `io::Write`. I could add some comments to make it clearer.

Reply:
If you want to. Probably what you said right here is sufficient :) feel free to merge after